### DISCUSSION TITLE
2007/04/16/%E5%8F%91%E7%8E%B0%E4%B8%80%E4%B8%AA%E6%96%B0%E7%9A%84%E6%90%9C%E7%B4%A2%E7%BD%91%E7%AB%99

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#152] [Announcements]
# 2007/04/16/%E5%8F%91%E7%8E%B0%E4%B8%80%E4%B8%AA%E6%96%B0%E7%9A%84%E6%90%9C%E7%B4%A2%E7%BD%91%E7%AB%99

网站名叫“有道 Youdao”：http://www.yodao.com/

https://blog.qizhen.xyz/2007/04/16/%E5%8F%91%E7%8E%B0%E4%B8%80%E4%B8%AA%E6%96%B0%E7%9A%84%E6%90%9C%E7%B4%A2%E7%BD%91%E7%AB%99

**Comment by ruanqizhen** [comment]
之前博客下的留言：


晃晃悠悠
2007年05月6日 00:33 编辑
呵呵…这也不失为推广的一种小手段
如同这个：根据你的blog测试你的性别
http://dingdang.spaces.live.com/blog/cns!C758121F778D6BDA!922.entry

回复
永青
2007年04月16日 21:14 编辑
这个网站不错啊
在us都不忘更新blog，赞的

回复
